container/functional/keylime_agent_container-basic-attestation/main.fmf

```diff
@@ -0,0 +1,29 @@
+summary: Tests basic keylime attestation scenario for keylime agent container
+description: |
+    Running only verifier and registrar on localhost. Agents run in containers.
+    Uses certificates generated by keylime.
+    Starts verifier, registrar on localhost.
+    Build container image with agent. 
+    Setup agent conf for each container.
+    Run container with agent.
+    Resgister agent by verifier.
+    Verifiers that container with agent passed attestation.
+    Does changes on a system and verifies that one container system has failed attestation. Second container again passed attestation.
```

Comment:
Do changes in scripts monitored by agents and confirm they fail attestation accordingly.